files/common/var/lib/delphix-platform/ansible/10-delphix-platform/roles/delphix-platform/tasks/main.yml

```diff
@@ -73,6 +73,40 @@
     state: directory
     mode: 01777
 
+#
+# Restrict cron permissions. All jobs are owned by root so there's no
+# reason to allow others any level of access.
```

Comment:
Can you make reference to the CIS benchmark here so that it's clear that this was added for a reason (in case someone feels like this is needless and feels compelled to remove it).